DayrSuryan/DSEthiop1.xml

```diff
@@ -561,6 +572,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <note>The legend is written in a crude hand, the vowel marker of <foreign xml:lang="gez">ዝ</foreign> is placed wrongly
                         (at the right leg of the letter, and turned to the right).</note>
                      </decoNote>
+                        
+                  <!--   <decoNote xml:id="drawing" corresp="#d3" xml:lang="ar">
```

Comment:
the decoration is the same, you need 2 desc one for each xml:lang, but the decoNote should be one, I think

Reply:
Same for the locus? For example:
```
    <locus target="#118v"/>
    <locus target="#118v" xml:lang="ar">ورقة ١١٨ ظ</locus>
```

Reply:
Is it for the Arabic inventory?

Reply:
exactly